manager/runner.go

```diff
@@ -1021,23 +1016,19 @@ func (r *Runner) diffAndUpdateDeps(depsMap map[string]dep.Dependency) {
 }
 
 // TemplateConfigFor returns the TemplateConfig for the given Template
-func (r *Runner) templateConfigsFor(tmpl *template.Template) []*config.TemplateConfig {
+func (r *Runner) templateConfigsFor(tmpl *template.Template) *config.TemplateConfig {
```

Comment:
Would it help to change this to `templateConfigFor` now that the map returns only one entry?

Reply:
Good idea... I'll update that now. Thanks!